### PR TITLE
Create Dataman Sales Screen UI

### PR DESCRIPTION
**Pull request by sameera222**
- [x] Design Sales Screen UI 
- [x] Add Extra Work Entry 
- [x] Add Unit Cancellation
- [x] Design Check Entry Page UI

### CODE REVIEW

File: apps/Dataman/src/components/listComponents/ChequeList.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+export const RECEIVE_LIST = [
```
Comment (sutyum) [inline apps/Dataman/src/components/listComponents/ChequeList.tsx]: why is this .tsx and not just .ts?

File: apps/Dataman/src/components/listComponents/EntryList.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+export const ENTRY_LIST = [
```
Comment (sutyum) [inline apps/Dataman/src/components/listComponents/EntryList.tsx]: why is this .tsx and not just .ts?

File: apps/Dataman/src/components/listComponents/UnitList.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+export const UNIT_LIST = [
```
Comment (sutyum) [inline apps/Dataman/src/components/listComponents/UnitList.tsx]: why is this .tsx and not just .ts?

**Review by sutyum — CHANGES_REQUESTED** [review]

File: apps/Dataman/src/components/listComponents/ChequeList.tsx (hunk 1 above)
Comment (sameera222) [reply apps/Dataman/src/components/listComponents/ChequeList.tsx]: refactored it

**Review by sutyum — APPROVED** [review]

File: apps/Dataman/src/components/listComponents/EntryList.tsx (hunk 2 above)
Comment (sameera222) [reply apps/Dataman/src/components/listComponents/EntryList.tsx]: refactored it

File: apps/Dataman/src/components/listComponents/UnitList.tsx (hunk 3 above)
Comment (sameera222) [reply apps/Dataman/src/components/listComponents/UnitList.tsx]: refactored the code

**Pull request merged**